web/src/pygments_data.ts

```diff
@@ -0,0 +1,7 @@
+import generated_pygments_data from "../generated/pygments_data.json";
+
+const langs: Record<string, {priority: number; pretty_name: string} | undefined> = generated_pygments_data.langs;
```

Comment:
It is probably worth giving the type that has `{priority: number; pretty_name: string}` a name `PygmentsLanguage`.